srcpkgs/lzfse/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'lzfse'
+pkgname=lzfse
+version=1.0
+revision=1
+build_style=cmake
+short_desc="LZFSE compression library and command line tool"
```

Comment:
let's split the library and devel from the cli tool

Reply:
Done.